docs/HowToGuides/FirstPullRequest.md

```diff
@@ -0,0 +1,168 @@
+  Since more people are likely to see and be able to answer your question, the
+  question is likely to get an answer sooner. Another benefit of asking in
+  public is that the answers you receive will be helpful to bystanders too.
+- Bug report/Pull request comments: Prefer asking in the bug report/pull request
+  when the question involves additional context specific to the
+  bug report/pull request.
+
+These are suggestions, not rules. For example, it's okay if you a broad question
+in a bug report or a pull request.
+
+When asking for help, prefer giving as much information as possible, while
+highlighting the parts that you think are important.
+
+### I didn't get a response from someone. What should I do?
+
+It's possible that you ask someone a question in a bug report/pull request and
+you don't get a response as quickly as you'd like. Maybe they are juggling
+several tasks and the discussion with you accidentally slipped by. Maybe they
+are on vacation or on leave for some reason. If you don't get a response
+within a week, it's okay to politely ping them using an `@` mention with a
+reminder. If you don't get a response for 2-3 weeks in a row, please ping
+someone else.
+
+## Working on a change
+
+Please see our [Getting Started guide][] to understand how to build the code,
+make changes, run tests and debug issues.
+
+[Getting Started guide]: /docs/HowToGuides/GettingStarted.md
+
+## Submitting a pull request
+
+### Tidying up
+
+Alright! You've implemented a change and would like to submit it.
+Double-check that you've tidied your Git history, such as squashing
+work-in-progress commits, and that your commit messages provide context.
+For example, if a commit fixes a bug, then include a "Fixes SR-NNNNN" with the
+bug number in the commit message.
+
+Next, [format your changes](/docs/HowToGuides/FAQ.md#how-do-i-format-my-changes)
+using `clang-format`.
+
+### Pushing and creating a pull request
+
+Assuming you followed the steps in our [Getting Started guide][], you should now
+be able to push your latest changes to GitHub using `git push`.
+
+Next, [create a pull request][] (PR). Usually, if you navigate to
+https://github.com/apple/swift right after pushing your change, GitHub will
+show a helpful "Compare & Pull Request" button.
+
+![Compare & Pull Request button in GitHub UI](/docs/GitHubCreatePRScreenshot.png)
+
+[create a pull request]: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request#creating-the-pull-request
+
+## Asking for code review
+
+If you had an active discussion with someone on how to implement your change,
+you should ask them for code review. If you directly implemented the change
+without any guidance from anyone else, you could select someone from GitHub's
+suggested reviewers. Please ask someone though! We don't want you to get stuck
+because you were not sure who to ask for code review.
```

Comment:
IIRC, you also need commit access to add reviewers, so new contributors will need to @ a committer and ask them to add themselves as a reviewer, or to select reviewers for them.

Reply:
Wow, I didn't realize that. https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/requesting-a-pull-request-review

> Note: Pull request authors can't request reviews unless they are either a repository owner or collaborator with write access to the repository.

I will update this.

Reply:
Yeah, Brent is correct that those without commit access (😇) cannot request reviewers.